.travis.yml

```diff
@@ -9,6 +9,7 @@ python:
 - '3.6-dev'
 env:
 - TOXENV=py-test
+- SELFINSTALL=1
```

Comment:
the correctly created sdist of tox is a problem, thus i choose to make different travis level calls